games-fps/sauerbraten/sauerbraten-2020.12.29.ebuild

```diff
@@ -36,13 +36,15 @@ BDEPEND="virtual/pkgconfig"
 
 PATCHES=(
 	# Respect CXXFLAGS, LDFLAGS
-	"${FILESDIR}"/${PN}-2020.12.27-respect-FLAGS-don-t-strip-symbols.patch
+	"${FILESDIR}"/${PN}-2020.12.29-respect-FLAGS-don-t-strip-symbols.patch
 
 	# Patch Makefile to use system enet instead of bundled
-	"${FILESDIR}"/${PN}-2020.12.27-unbundle-enet.patch
+	"${FILESDIR}"/${PN}-2020.12.29-unbundle-enet.patch
 
 	# Don't use freetype-config, it's obsolete
-	"${FILESDIR}"/${PN}-2020.12.27-use-pkg-config-for-freetype2.patch
+	"${FILESDIR}"/${PN}-2020.12.29-use-pkg-config-for-freetype2.patch
+	# More sensible ways of including SDL_mixer and SDL_image. Game doesn't build w/o this.
+	"${FILESDIR}"/${PN}-2020.12.29-includefix.patch
```

Comment:
Was this upstreamed? If not, it feels like something that should be send upstream, then add a link to upstream the issue/pr. Otherwise comment with a link to the upstream change that introduces it.

Reply:
Okay, I will look into upstreaming this.

Reply:
Hi, this project uses sourceforge and it doesn't have a fork button. I guess I can't upstream this. But I will fix all the other issues you listed above.

Reply:
Yeah, it looks like they don't even have a normal bug tracker, and you have to use the forum. Oh well. Let's leave it unless you feel like doing it.